service/src/main/kotlin/fi/espoo/evaka/messaging/message/MessageControllerCitizen.kt

```diff
@@ -79,6 +86,16 @@ class MessageControllerCitizen(
         return db.read { it.getMessagesSentByAccount(accountId, pageSize, page) }
     }
 
+    @GetMapping("/receivers")
+    fun getReceivers(
+        db: Database.Connection,
+        user: AuthenticatedUser,
+    ): List<MessageAccount> {
+        val accountId = requireMessageAccountAccess(db, user)
+        Audit.MessagingCitizenFetchReceiversForAccount.log(targetId = accountId)
```

Comment:
Audit logging needs to happen first, before `requireMessageAccountAccess`. Otherwise some requests are not logged if errors are thrown from the function that is called first.